src/utils/formContext.ts

```diff
@@ -47,6 +47,7 @@ export const getFormContext = (formUuid: string) => ({
       : {};
     return {
       totalSteps: Object.keys(state.steps).length,
+      stepId: step.id ?? '',
```

Comment:
why is stepId being returned in the form context? this should be an internal identifier

Reply:
The steps filter on logicRule is an array of step ids.  Step ids are better than names/keys which can change.  So I need the step id here.  We need not document it so it stays internal.  See runUserLogic where I filter by any steps and use this.

Reply:
Ok, made this change